third-party/CMakeLists.txt

```diff
@@ -12,8 +12,10 @@ add_subdirectory(fmt EXCLUDE_FROM_ALL)
 # MagicEnum
 add_subdirectory(magic_enum EXCLUDE_FROM_ALL)
 
+if(NOT ENABLE_QT_GUI)
```

Comment:
SDL shouldn't be disabled with Qt, we may want to use it for input for example

Reply:
i did got a weird error , we will re-enable it once it will be useful for qt frontend